debug/TorrentClient/test_get_message.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from message import Message, \
+                    KeepAlive, \
+                    Choke, \
+                    UnChoke, \
+                    Interested, \
+                    NotInterested, \
+                    Have, \
+                    BitField, \
+                    Request, \
+                    Piece, \
+                    Cancel, \
+                    Port
+
+class TestGetMessage(unittest.TestCase):
+
+    def test_keep_alive(self):
+    	message = Message.get_message('keep-alive')
+    	self.assertEqual(message, KeepAlive())
+
+    def test_choke(self):
+    	message = Message.get_message('choke')
+    	self.assertEqual(message, Choke())
+
```

Comment:
and this would be choke:
 def test_get_message_choke(self):

       msg = Message.get_message('choke')

       self.assertEqual(isinstance(msg, Choke), True)



   def test_get_message_choke_length(self):

       msg = Message.get_message('choke')

       self.assertEqual(msg.length, 1)



   def test_get_message_choke_id(self):

   		msg = Message.get_message('choke')

   		self.assertEqual(msg.id, 0)

since you need length and id, not just length like in keep-alive